gh_pages/templates/js/docs.js

```diff
@@ -0,0 +1,34 @@
+CodeMirror.defineMode('mustache', function(config, parserConfig) {
+  var mustacheOverlay = {
+    token: function(stream) {
+      var ch;
+      if (stream.match('{{')) {
+        while ((ch = stream.next()) != null)
+          if (ch == '}' && stream.next() == '}') {
+            stream.eat('}');
+            return 'mustache';
+          }
+      }
+      while (stream.next() != null && !stream.match('{{', false)) {
+      }
+      return null;
+    }
+  };
+  return CodeMirror.overlayMode(CodeMirror.getMode(config, parserConfig.backdrop || 'text/html'), mustacheOverlay);
+});
+
+window.setTimeout(function() {
+  _.each(document.querySelectorAll('pre > code'), function(el) {
+    var code = el.textContent;
+    var mode = el.className ? el.className.substring(5) : 'javascript';
```

Comment:
this bit seems a bit fragile.. maybe a data-attribute instead of substringing className? or maybe:
```
if (el.classList.contains('html')) {
  mode = 'mustache';
} else ..
```